presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergSplit.java

```diff
@@ -26,6 +26,7 @@
 import java.util.Collections;
 import java.util.List;
 import java.util.Map;
+import java.util.Objects;
```

Comment:
Error:  src/main/java/com/facebook/presto/iceberg/IcebergSplit.java:[29,8] (imports) UnusedImports: Unused import - java.util.Objects.

Reply:
Thanks for remind me this style issue, I revert the IcebergSplit change since I'm not sure how to change the hash function for IcebergSplit, for HiveSplit, we tested the changed hash function can gain better performance.